doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -325,7 +351,7 @@ NOTE: `curl` is a Unix command allowing for interacting with REST APIs such as t
 | enable\_encoding | no | false | <i>true</i> or <i>false</i>, <i>true</i> applies the new encoding, <i>false</i> applies the old encoding. ||
 | enable\_grouping | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_grouping_interceptor.md) for more details. ||
 | enable\_name\_mappings | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_name_mappings_interceptor.md) for more details. ||
-| data\_model | no | dm-by-entity |  Always <i>dm-by-entity</i>, even if not configured. ||
+| data\_model | no | dm-by-entity | <i>dm-by-entity-id</i>, <i>dm-by-entity</i> ||
```

Comment:
Previously to this PR, dm-by-entity was the only mode supported, wasn't it? Or, in other words, this PR has added the possibility of using dm-by-entity-id.

Is my understanding correct?

Reply:
Yes , its correct

Reply:
Thanks for the clarification!

NTC